ui/guest/src/classes/ContentController.ts

```diff
@@ -102,67 +108,68 @@ export class ContentController {
 
   }
 
-  getModel(modelId)/*: Promise<Model> */ {
+  getModel(modelId: string): Promise<ContentInstance> {
     return this.getModel$(modelId).toPromise();
   }
 
-  getModel$(modelId)/*: Observable<Model> */ {
+  getModel$(modelId: string): Observable<ContentInstance> {
     return this.models$(modelId).pipe(
       filter(models => modelId in models),
       map(models => models[modelId])
     );
   }
 
-  getContentType(contentTypeId) {
+  getContentType(contentTypeId: string): Promise<ContentType> {
     return this.getContentType$(contentTypeId).toPromise();
   }
 
-  getContentType$(contentTypeId) {
+  getContentType$(contentTypeId: string): Observable<ContentType> {
     (!this.hasCachedContentType(contentTypeId)) && this.fetchContentType(contentTypeId);
     return ContentController.contentTypesObs$.pipe(
       filter(contentTypes => contentTypeId in contentTypes),
       map(contentTypes => contentTypes[contentTypeId])
     );
   }
 
-  models$(modelId)/*: Observable<Model> */ {
+  models$(modelId?: string): Observable<LookupTable<ContentInstance>> {
     (modelId && !this.hasCachedModel(modelId)) && this.fetchModel(modelId);
     return ContentController.modelsObs$;
   }
 
-  contentTypes$()/*: Observable<Model> */ {
+  contentTypes$(): Observable<LookupTable<ContentType>> {
     return ContentController.contentTypesObs$;
   }
 
-  operations$() {
-    return ContentController.operationsObs$;
-  }
+  // TODO: implement operationsObs$
+  // operations$() {
+  //   return ContentController.operationsObs$;
+  // }
```

Comment:
??

Reply:
@rart after merge the method was there but there is no operationsObs$ defined. Should it be removed or implemented?

Reply:
Remove.